src/app/components/practice-container/practice-container.component.sass

```diff
@@ -0,0 +1,11 @@
+.practice-container
+    justify-content: center
+    align-content: center
```

Comment:
I think this only has an effect if the parent container is a flexbox, which I think is not the case (see Line 4 of app.component.html)